CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 1.5.0 - 18-OCT-2019
```

Comment:
[1.4.1 was not released on RubyGems](https://github.com/onesky/onesky-rails/issues/29).
Could you please make sure 1.5.0 will? 🙏🏻